src/app/services/navigation/navigation.service.ts

```diff
@@ -43,7 +43,8 @@ export class NavigationService {
       name: T('Datasets (WIP)'),
       type: MenuItemType.Link,
       tooltip: T('Datasets'),
-      icon: 'dns',
+      icon: 'dataset',
```

Comment:
I left a [comment](https://ixsystems.atlassian.net/browse/NAS-117317?focusedCommentId=152138) under the ticket. I'm okay with keeping it

Reply:
Implemented as an ix-icon tag